### PR TITLE
Add 1.13.1 and 1.13.2 to the docs

### PR DESCRIPTION
**Pull request by hhunter-ms**
#4109

### CODE REVIEW

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ The table below shows the versions of Dapr releases that have been tested togeth
 
 | Release date | Runtime     | CLI  | SDKs  | Dashboard  | Status | Release notes |
 |--------------------|:--------:|:--------|---------|---------|---------|------------|
+| April 3rd 2024 | 1.13.2</br>  | 1.13.0 | Java 1.11.0 </br>Go 1.10.0 </br>PHP 1.2.0 </br>Python 1.13.0 </br>.NET 1.13.0 </br>JS 3.3.0 | 0.14.0 | Supported (current) | [v1.13.0 release notes](https://github.com/dapr/dapr/releases/tag/v1.13.0) |
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-release-policy.md]:
```suggestion
| April 3rd 2024 | 1.13.2</br>  | 1.13.0 | Java 1.11.0 </br>Go 1.10.0 </br>PHP 1.2.0 </br>Python 1.13.0 </br>.NET 1.13.0 </br>JS 3.3.0 | 0.14.0 | Supported (current) | [v1.13.2 release notes](https://github.com/dapr/dapr/releases/tag/v1.13.2) |
```

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ The table below shows the versions of Dapr releases that have been tested togeth
...
+| March 26th 2024 | 1.13.1</br>  | 1.13.0 | Java 1.11.0 </br>Go 1.10.0 </br>PHP 1.2.0 </br>Python 1.13.0 </br>.NET 1.13.0 </br>JS 3.3.0 | 0.14.0 | Supported (current) | [v1.13.0 release notes](https://github.com/dapr/dapr/releases/tag/v1.13.0) |
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-release-policy.md]:
```suggestion
| March 26th 2024 | 1.13.1</br>  | 1.13.0 | Java 1.11.0 </br>Go 1.10.0 </br>PHP 1.2.0 </br>Python 1.13.0 </br>.NET 1.13.0 </br>JS 3.3.0 | 0.14.0 | Supported (current) | [v1.13.1 release notes](https://github.com/dapr/dapr/releases/tag/v1.13.1) |
```

**Review by msfussell — COMMENTED** [review]
Couple of updates

**Review by msfussell — APPROVED** [review]
LGTM

**Comment by marcduiker** [comment]
@holopin-bot @hhunter-ms Thanks Hannah!

**Pull request merged**